src/Core/Geographic/Crs.js

```diff
@@ -2,38 +2,43 @@ import proj4 from 'proj4';
 
 proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
 
-const TMS = [
-    'WMTS:WGS84',
-    'WMTS:PM',
-];
+const root_tms = 'TMS';
+const root_epsg = 'EPSG';
+const nCrs = ['4326', '3857'];
 
-const EPSG = [
-    'EPSG:4326',
-    'EPSG:3857',
-];
+const TMS = nCrs.map(n => `${root_tms}:${n}`);
+const EPSG = nCrs.map(n => `${root_epsg}:${n}`);
```

Comment:
Too complicated imho, I understand what you are aiming for, but I see no problem in using already completed array. Same goes for `root_`, use directly the string: it won't change any time soon, so no worry about maintainability.

Reply:
ok, I removed `root_`